qt/aqt/webview.py

```diff
@@ -248,6 +248,7 @@ class AnkiWebViewKind(Enum):
     EMPTY_CARDS = "empty cards"
     FIND_DUPLICATES = "find duplicates"
     FIELDS = "fields"
+    IMPORT_LOG = "import log"
```

Comment:
Presumably this will change to "apkg import"

Reply:
The page is shown for JSON imports too, so I kept this as it's.